helm/linkerd2-multicluster-app/values.yaml

```diff
@@ -54,6 +54,3 @@ enablePSP: true
 # across hosts and zones for High Availability.
 # Enable this only when you have multiple replicas of components.
 enablePodAntiAffinity: true
-
-# -- User ID for the gateway
-controllerUID: 2103
```

Comment:
does ti work without controllerUID?

Reply:
It is not part of upstream [2.11.4](https://github.com/linkerd/linkerd2/blob/stable-2.11.4/multicluster/charts/linkerd-multicluster/values.yaml) and they changed it to a different key for [2.12.0](https://github.com/linkerd/linkerd2/blob/stable-2.12.0/multicluster/charts/linkerd-multicluster/values.yaml#L30)..